src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import {Calculator} from './components/Calculator'
+import './index.css'
+
+
+
+const App = ()=>{
+  return (
+    <Calculator/>
+  )
+}
```

Comment:
[OPTIONAL] You have done a very good job on this project. Your code is well organized 👍. I would only advise that you develop the habit of keeping the app component inside its own file. The app component tends to be useful for many other implementations and can get pretty large when you scale your projects up. Of course, in this case, this way works just fine but your code will sure be cleaner if you keep the App component in its own file or module and import it into the index.js file 😉